src/PrestaShopBundle/Form/Admin/Sell/Order/Invoices/InvoiceOptionsType.php

```diff
@@ -79,35 +91,72 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('enable_invoices', SwitchType::class)
-            ->add('enable_tax_breakdown', SwitchType::class)
-            ->add('enable_product_images', SwitchType::class)
-            ->add('invoice_prefix', TranslatableType::class, [
+            ->add(static::FIELD_ENABLE_INVOICES, SwitchType::class, [
```

Comment:
I don't see the interest in storing these fields into constants here We never do that in any form Besides the constants are not used somewhere else so they lose all their interest

Reply:
I think I wanted to use constants when getting label but found another way. Will remove this.